### PR TITLE
Update Compile-from-Source Instructions

### PR DESCRIPTION
**Pull request by spomorski**
Updating CfS instructions together with David.

### CODE REVIEW

**Comment by iirina** [comment]
Are all commits reviewed?

File: site/docs/install-windows.md [hunk 1]
```diff
@@ -9,23 +9,52 @@ Supported Windows platforms:
...
 *   [Download the binary (recommended)](#download-the-binary-recommended)
 *   [Install using Chocolatey](#install-using-chocolatey)
 *   [Compile Bazel from source](install-compile-source.html)
 
-## Download the binary (recommended)
+#### Download the binary (recommended)
 
 Go to Bazel's [GitHub releases page](https://github.com/bazelbuild/bazel/releases)
-and download the Windows binary.
+and download the Windows binary<sup>1</sup>: `bazel-<version>-installer-windows-x86_64.sh`.
```
Comment (laszlocsomor) [inline site/docs/install-windows.md]: There's no installer on Windows. The user downloads a ready-to-run binary.
Please remove this information along with the footnote.

To expedite submitting this PR, I'll import it into the internal version control and make this edit myself, then send it to @spomorski .

**Comment by laszlocsomor** [comment]
I tried importing this change and resolving the conflicts, but it seems that parts of this PR have already been merged, e.g. install-ubuntu.md shows the new version and install-windows shows some of the changes (e.g. the `<sup>1</sup>` part).
Because of that and because some information is incorrect (e.g. on the Windows page; see my review comment) and because I want to simplify the Windows install page, I suggest abandoning this PR.

**Comment by spomorski** [comment]
Closed.

**Pull request closed without merging**